packages/itwinui-variables/src/themes/dark-hc.scss

```diff
@@ -151,8 +150,8 @@ $opacities: (
   --iui-color-text-negative-hover: #{hex($fg-negative-0)};
   // #endregion
   // #region HSL variables for transparency
-  --iui-color-background-hsl: #{get-hsl($bg-6)};
-  --iui-color-foreground-hsl: #{get-hsl($fg-1)};
+  --iui-color-background-hsl: #{get-hsl($bg-500)};
```

Comment:
I think bg-6 should be converted to bg-550?

Reply:
`--iui-color-background-hsl` should match whatever is being used by `--iui-color-background`, so for `dark-hc.scss` `$bg-500` appears to be correct.

Reply:
previous mistake